hack/update-bazel.sh

```diff
@@ -17,6 +17,9 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+export LANG=C
+export LC_ALL=C
```

Comment:
elsewhere in the code base we set this inline:
https://github.com/kubernetes/kubernetes/blob/master/hack/lib/util.sh

Reply:
I don't think we should handle this particular script differently, it _might_ make sense to globally export these in the hack/lib, but that might need some discussion from the other maintainers, I'd just set LC_ALL=C in the pipeline below

Reply:
Yeah, adding it globaly in `util.sh` is the best solution in the long run, as long as all scripts include it. 

Guess i can move it down into the sort cmd again if that is preferred. 

In `hack/update-vendor.sh`, it is exported in the top:

https://github.com/kubernetes/kubernetes/blob/4aaa7b4da2643d5085d1844495c0d6cff6970cf5/hack/update-vendor.sh#L28

Kinda annoying that some scripts dont sort stable (like this), so that I always have to diff stuff manually. :/ I dont see any reason _not_ to use `LC_ALL=C LANG=C` and sort stable on all platforms & locales...

I guess all the CI scripts run with `LC_ALL=C LANG=C`, so adding it to the util.sh-file shouldn't make any different.

Reply:
Moved them inline again now :smile:

Reply:
thanks!